scrapers/KristenBjorn.yml

```
@@ -0,0 +1,114 @@
+name: "KristenBjorn"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/video
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/web/model/store/
+    scraper: movieScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - kristenbjorn.com/gay-porn-star/
+      - kristenbjorn.com/web/model/gay-porn-star/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //div[@class="scene-item"]/a[@class="scene-box"]
+    scene:
+      Title: //article/div[1]//h2/text()
+      Details: //div[@class="med-text"]/p/text()
```

There are multiple `text()` in some `<p>` due to some `<span>`s
Doing something like this
```yaml
      Details:
        selector: //div[@class="med-text"]/p/text()
        concat: " "
```

or even better 
```suggestion
      Details: //div[@class="med-text"]/p
``` 
is needed to get all of  the description